src/ontology/uberon-edit.obo

```diff
@@ -11741,13 +11741,12 @@ property_value: homology_notes "The bilaterian gut is typically a complete tube
 [Term]
 id: UBERON:0001047
 name: neural glomerulus
-def: "Large synaptic bouton found in areas like olfactory bulb, cerebellar cortex and thalamus, that contacts many post-synaptic targets[NIF]." [http://ccdb.ucsd.edu/SAO/images/glomerulus_sw_512x512.jpg, http://flybrain.uni-freiburg.de/Flybrain/html/terms/terms.html]
-comment: isa or partof neuropil? TODO. Add to GO-CC?
+def: "Large synaptic bouton found in areas like olfactory bulb, cerebellar cortex and thalamus, that contacts many post-synaptic targets[NIF]." [DOI:10.1016/B978-0-12-374245-2.00007-3]
 subset: uberon_slim
 synonym: "glomerulus" BROAD [FBbt:00005386, NIF_Subcellular:sao587858733]
 xref: NIF_Subcellular:sao587858733
```

Comment:
```suggestion
xref: ILX:0728797
```

Reply:
NIF_Subcellular xref doesnt resolve, and NIF_Subcellular prefixes eventually resolves to ILX anyway, so this is probably more appropriate an xref regardless